### ISSUE TITLE
Make it possible to run more than one mjpeg omxplayer sessions in parallel

### ISSUE THREAD

**Issue by shuckle50** [huceke/omxplayer#163]
Details are in 
https://github.com/huceke/omxplayer/issues/27#issuecomment-16718570

Just filed this as this is separate issue and easily forgotten without a real issue doc.

**Comment by Grunge** [comment]
Push up.
Jup, make it possible please.

**Comment by popcornmix** [comment]
I believe this is fixed in latest firmware. Try rpi-update.

**Comment by shuckle50** [comment]
Thanks a lot !!
Works nicely (but indeed required git-update).
